src/media/css/pageheader.styl

```diff
@@ -0,0 +1,34 @@
+.pageheader
+  align-items flex-end
+  border-bottom 1px solid #DDD
+  display flex
+  justify-content space-between
+  
+  h1
+    font-size 28px
+    font-weight 400
+    margin 0
+
```

Comment:
2 lines for first-level indentation groups, 1 line for everything else? I'm all aboard on that :bullettrain_side: